pom.xml

```diff
@@ -75,7 +75,7 @@
         <airline.version>2.2.0</airline.version>
         <antlr.version>4.7</antlr.version>
         <csv.version>1.4</csv.version>
-        <guava.version>21.0</guava.version>
+        <guava.version>24.0-jre</guava.version>
```

Comment:
what is this `-jre` suffix?

Reply:
Guava 24.0 has two published releases:  "-android" for the Android platform.  "-jre" for Oracle Java 8 and above.

Reply:
see:  https://github.com/google/guava